Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -12,6 +12,12 @@
     2. Escalating abnormal influx of bugs, so the release, as a whole, can be reassessed.
 4. Making sure they can dedicate enough time for the release. In case this is not possible (due to other squad commitments), please inform the iOS chapter lead.
 
+**The objective is to ship the release build as soon as possible whilst maintaining the quality bar and addressing every bug raised by QA**. When dealing with a particularly complicated bug (one that would require a rather significant effort to address) the release engineer should speak with either Andreaa Papillon (Native Apps' Product Manager) or Lilla Szulyovszky (Native Apps' Delivery Manager) as every release is managed and signed-off by the Native Apps squad. The bug's corresponding feature should disabled altogether using its feature switch, if applicable. Such a bug would subsequently be handled by its respective squad.
```

Comment:
`The bug's corresponding feature should disabled altogether using its feature switch`
->
`The bug's corresponding feature should be disabled altogether using its feature switch`